### ISSUE TITLE
[REQUEST] Use vectorbased coloring of regions instead of a marker

### ISSUE THREAD

**Issue by Ave-Rechts** [seanmorley15/AdventureLog#335]
**Is your feature request related to a problem? Please describe.**
When u mark a region as visited and u go look on the map, it is shown as a single marker. This doesn't really showcase properly where this region is situated.

**Describe the solution you'd like**
When a region is visited and u want to see these on the world map, the entire region should be filled with a 50% opacity or something. This makes it more clear, more visually appealing as well.

**Additional context**
Had a small discussion with Sean already. We probably need to find a GeoJSON db somewhere that already has this information. No need to reinvent the wheel on this data.

**Comment by larsl-net** [comment]
I played a little bit around with the GeoJSON feature of svelte-maplibre.
I use the OpenStreet map Overpass API to generate the GeoJSON data.

[Code](https://github.com/larsl-net/AdventureLog/tree/region-color)

[PoC to generate GeoJSON](https://github.com/larsl-net/AdventureLog-GeoJSON)

![Image](https://github.com/user-attachments/assets/d6938000-a8d9-4e29-8f9c-faf85a1ddb7c)

**Comment by seanmorley15** [comment]
This is awesome @larsl-net, is it ok if I use some of the code from https://github.com/larsl-net/AdventureLog-GeoJSON/tree/main to add it into the seeding command on startup to generate the file. I am not sure if this is the best way I'm just going to play around with some f the stuff you provided here - thanks a lot!!

**Comment by larsl-net** [comment]
@seanmorley15 yes you can use this.

Maybee is is better to generate the data central and deliver as separate image or publishing over a CDN. This should reduce the request/load which is generated on the overpass API.

**Comment by seanmorley15** [comment]
@larsl-net yes this is absolutely a good idea!! Now that I have an AdventureLog cloud server I could totally do something like this!

**Comment by seanmorley15** [comment]
@larsl-net, I am planning to make a simple docker container that has python and nginx. This will run a script periodically to fetch the newest data and serve it. I will host this server but I will make it so it can be overridden with a custom URL in AdventureLog for local hosting.
